ghiscoding/slickgrid-react
# fix(comp): switch to reusable Slickgrid-Universal Pagination

- Pagination is now a reusable Slickgrid-Universal pagination, used by 2 libs, (all written in plain TypeScript) is well tested and handles all scenarios

First Contribution which has the following
1. upgrade to latest Slickgrid-Universal versions
2. remove internal pagination component 
3. use Slickgrid-Universal pagination instead. 
4. Also while I was at it, I took a quick peak at the code in Example 1 and applied it to Example 2 and it fixed that demo as well, see for yourself below 😉 
 
cheers

![2021-11-16_22-03-58](https://user-images.githubusercontent.com/643976/142127855-915846fb-5189-4c35-a3c2-61dabdbb6c14.gif)
